### PR TITLE
Add GPy project description

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

File: _project_descriptions/gpy.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<embed src="http://gpss.cc/gpss17/slides/multipleOutputGPs.pdf#page=4" type="application/pdf">
+
+A Gaussian process model can be created that allows prediction, with a measure of uncertainty, of any of the measurements at any point in the space. Furthermore, [inference of underlying functions driving the measurements](http://proceedings.mlr.press/v5/alvarez09a/alvarez09a.pdf) is also possible.
+
+[GPy](https://github.com/SheffieldML/GPy) is a popular framework for Gaussian processes in written in python. However, code to execute advanced multiple output Gaussian processes was only available in [MatLab](https://uk.mathworks.com/products/matlab.html), which is less widely used for Gaussian processes and less well suited as a development platform for this work.
+
+The RSE involvement was in documenting aspects of GPy and converting MatLab code into reliable python in the GPy framework. We used tests to compare code python code output to a MatLab baseline as a means of driving development.
```
Comment (willfurnass) [inline _project_descriptions/gpy.md]: Could something about architectural docs being of value for sustaining GPy?

File: _project_descriptions/gpy.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Boarder access to multiple output Gaussian process modelling is of potential benefit to a range of fields and activities. Examples include pollution modelling, robotics, gene regulation and financial services.
```
Comment (willfurnass) [inline _project_descriptions/gpy.md]:
```suggestion
Broader access to multiple output Gaussian process modelling is of potential benefit to a range of fields and activities. Examples include pollution modelling, robotics, gene regulation and financial services.
```

File: _project_descriptions/gpy.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+key: gpy
+---
+
+Multiple output Gaussian processes are useful where several different measurements are made at different points in a parameter space, but not all measurements are made at each point. For example:
+
+<embed src="http://gpss.cc/gpss17/slides/multipleOutputGPs.pdf#page=4" type="application/pdf">
+
+A Gaussian process model can be created that allows prediction, with a measure of uncertainty, of any of the measurements at any point in the space. Furthermore, [inference of underlying functions driving the measurements](http://proceedings.mlr.press/v5/alvarez09a/alvarez09a.pdf) is also possible.
+
+[GPy](https://github.com/SheffieldML/GPy) is a popular framework for Gaussian processes in written in python. However, code to execute advanced multiple output Gaussian processes was only available in [MatLab](https://uk.mathworks.com/products/matlab.html), which is less widely used for Gaussian processes and less well suited as a development platform for this work.
```
Comment (willfurnass) [inline _project_descriptions/gpy.md]:
```suggestion
[GPy](https://github.com/SheffieldML/GPy) is a popular framework for Gaussian processes in written in Python. However, code to execute advanced multiple output Gaussian processes was only available in [MATLAB](https://uk.mathworks.com/products/matlab.html), which is less widely used for Gaussian processes and less well suited as a development platform for this work.
```

File: _project_descriptions/gpy.md (hunk 1 above)
Comment (willfurnass) [inline _project_descriptions/gpy.md]:
```suggestion
The RSE involvement was in documenting aspects of GPy and converting MATLAB code into reliable Python in the GPy framework. We used tests to compare code python code output to a MATLAB baseline as a means of driving development.
```

File: _project_descriptions/gpy.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+key: gpy
+---
+
+Multiple output Gaussian processes are useful where several different measurements are made at different points in a parameter space, but not all measurements are made at each point. For example:
+
+<embed src="http://gpss.cc/gpss17/slides/multipleOutputGPs.pdf#page=4" type="application/pdf">
+
+A Gaussian process model can be created that allows prediction, with a measure of uncertainty, of any of the measurements at any point in the space. Furthermore, [inference of underlying functions driving the measurements](http://proceedings.mlr.press/v5/alvarez09a/alvarez09a.pdf) is also possible.
```
Comment (willfurnass) [inline _project_descriptions/gpy.md:9]: Worth adding anything about LTMAAH objectives inc coupling with PDE (ODE?) solver?

File: _data/projects.csv [hunk 5]
```diff
@@ -23,7 +23,7 @@ id,key,title,long_title,tech_methods,rses,start,end,department,level,show
 31,,Teaching Support for GPU Module,Teaching Support for GPU Module,,"Mozhgan Kabiri Chimeh, Robert Chisholm",01/02/2019,01/09/2021,Computer Science,2,1
 32,,SoBigData++,SoBigData++,,"David Jones, Twin Karmakharm",01/01/2020,01/01/2022,Computer Science,2,1
 33,,RISIS II (includes WeVerfiy and some ELG),RISIS II (includes WeVerfiy and some ELG),,"Twin Karmakharm, David Wilby",01/01/2020,30/06/2021,Computer Science,2,1
-34,,Learning to move as a human,Learning to move as a human,"python, Gaussian processes, documentation, testing, continuous integration",Robert Turner,01/10/2019,01/01/2021,Computer Science,2,1
+34,gpy,Learning to move as a human,Multiple output Gaussian processes for GPy,"python, Gaussian processes, documentation, testing, continuous integration, matlab",Robert Turner,01/10/2019,01/01/2021,Computer Science,1,1
```
Comment (willfurnass) [inline _data/projects.csv]:
```suggestion
34,gpy,Learning to move as a human,Multiple output Gaussian processes for GPy,"python, Gaussian processes, documentation, testing, continuous integration, MATLAB",Robert Turner,01/10/2019,01/01/2021,Computer Science,1,1

```

**Review by willfurnass — APPROVED** [review]
Noted some minor suggestions but LGTM.

File: _project_descriptions/gpy.md (hunk 1 above)
Comment (bobturneruk) [reply _project_descriptions/gpy.md]: Yes!

File: _project_descriptions/gpy.md (hunk 2 above)
Comment (bobturneruk) [reply _project_descriptions/gpy.md]: Thanks!

File: _project_descriptions/gpy.md (hunk 3 above)
Comment (bobturneruk) [reply _project_descriptions/gpy.md]: Yes!

File: _project_descriptions/gpy.md (hunk 1 above)
Comment (bobturneruk) [reply _project_descriptions/gpy.md]: Ta!

File: _project_descriptions/gpy.md (hunk 4 above)
Comment (bobturneruk) [reply _project_descriptions/gpy.md:9]: Could be that some more info. would be useful, but I don't think (fundamentally) we were solving differential equations. I'd have to properly get my head back into it to explain in more detail. Maybe later!

File: _data/projects.csv (hunk 5 above)
Comment (bobturneruk) [reply _data/projects.csv]: Sure. Waiting for MathWorks to re-standardise on M A T L A B ! ! !

**Comment by bobturneruk** [comment]
Thanks @willfurnass !

File: _data/projects.csv (hunk 5 above)
Comment (willfurnass) [reply _data/projects.csv]: Sorry for the pedantry! :)

Comment (willfurnass) [reply _data/projects.csv]: And I'm holding out them standardising on the more appropriate:
```
M L
A A
T B
```

Comment (bobturneruk) [reply _data/projects.csv]: Thank you for the pedantry!

Comment (bobturneruk) [reply _data/projects.csv]: Could be:

```
M A
T L
A B
```

I can't remember.

**Pull request merged**